workflows/variantinterpretation.nf

```diff
@@ -180,6 +197,18 @@ workflow VARIANTINTERPRETATION {
         ch_versions = ch_versions.mix(VEMBRANE_TABLE.out.versions)
     }
 
+    ///
+    // MODULE: TMB calculation
+    ///
+    if ( params.bedfile && params.calculate_tmb ) {
```

Comment:
This block should be moved into the tsv if-else block, as the TSV conversion is required.
Add to parameter check in beginning to give an error (with `error()`), if `tsv=false` and `calculate_tmb==true` is set.